great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -274,41 +274,54 @@ def _prepare_v1_config(cls, config: dict) -> dict:
         # to prevent downstream issues
         config["fluent_datasources"] = _extract_fluent_datasources(config)
 
-        # V1 renamed EvaluationParameters to SuiteParameters, so this is a temporary patch
-        # until Cloud implements a V1 endpoint for DataContextConfig
-        config = cls._replace_evaluation_parameters_with_suite_parameters(config)
+        # V1 renamed EvaluationParameters to SuiteParameters, and Validations to ValidationResults
```

Comment:
Hey Reviewer! Please give this an extra set of eyes!

Reply:
Could we use constants here for the names? I think we might have some of these as `DataContextVariables` or in `great_expectations/data_context/types/base.py`?

Reply:
Alternatively, should we change test code instead of source code here? I'm assuming we're making these changes to allow tests to pass right?

Reply:
AFAIK We need to do this mapping in the source code, at least for the fields not prefixed with "default", as the data coming back from cloud will have the old keys. But looking again, I think we can probably get rid of the renaming of "default_evaluation_parameter_store" and "default_validations_store", since I think those can be whatever.

I'll look into removing the stuff about "default_", and use the constants from `DataContextVariableSchema` for the v1 values (thanks for pointing me in that direction!).

I don't personally care too much about making constants for the old keys, since they are only used once, and I think their meaning should be pretty clear here. But if you feel strongly in the other direction, fell free to throw a blocker on here and make me do it 😄